src/cli/cli.cc

```diff
@@ -960,7 +960,7 @@ int main (const int argc, const char* argv[]) {
         }
 
         // default values
-        settings["build_output"] = settings["build_output"].size() > 0 ? settings["build_output"] : "dist";
+        settings["build_output"] = settings["build_output"].size() > 0 ? settings["build_output"] : "build";
```

Comment:
so now we have `"build"` by default in both `.gitignore` and as an implicit default value